python/port/mod/time/modtime.cpp

```diff
@@ -1,17 +1,21 @@
 extern "C" {
 #include "modtime.h"
+#include <py/runtime.h>
 }
 #include <ion/timing.h>
 #include "../../helpers.h"
 #include <py/smallint.h>
-#include <py/runtime.h>
 
 mp_obj_t modtime_sleep(mp_obj_t seconds_o) {
 #if MICROPY_PY_BUILTINS_FLOAT
-  micropython_port_interruptible_msleep(1000 * mp_obj_get_float(seconds_o));
+  mp_float_t tts = 1000 * mp_obj_get_float(seconds_o);
```

Comment:
I think `tts` should be declared as an `int32_t` since it is what `micropython_port_interruptible_msleep`eventually expects. Also `tts` should probably be defined outside of the #if block.

Reply:
Done with [`4de2bc3` (#1662)](https://github.com/numworks/epsilon/pull/1662/commits/4de2bc3367126e5403be94d2d7c3b10b8afdb749)